tests/tabular_tests/records_tests/test_lists_factory.py

```diff
@@ -49,3 +50,13 @@ def test_lists_factory(data: DataObject,
     assert all(map(partial(table_record_is_valid,
                            table=table),
                    result))
+
+    unique_indices = [
+        index for index, column in enumerate(table.columns)
+        if is_column_unique(column)
+    ]
```

Comment:
Not necessary since I moved it a separate function in `test.utils` according to https://github.com/lycantropos/hypothesis_sqlalchemy/pull/16#discussion_r302951640